frontend/src/components/ChartWidget.tsx

```diff
@@ -109,6 +109,7 @@ function ChartWidgetComponent(props: Props) {
           parts={keys}
           data={filteredJson}
           significantDigitLabels={content.significantDigitLabels}
+          computePercentages={content.computePercentages}
```

Comment:
Why to add this here if we did not change the PartToWhole component?

Reply:
Missed that when I was removing partofwhole